tests/chart_tests/test_byo_sa.py

```diff
@@ -43,12 +48,26 @@ def test_serviceaccount_with_overrides(self, kube_version):
                 "charts/astronomer/templates/registry/registry-serviceaccount.yaml",
                 "charts/astronomer/templates/config-syncer/config-syncer-serviceaccount.yaml",
                 "charts/astronomer/templates/houston/api/houston-bootstrap-serviceaccount.yaml",
+                "charts/astronomer/templates/astro-ui/astro-ui-serviceaccount.yaml",
```

Comment:
Can we change this list to a function that finds all `*serviceaccount*` templates? Like: 

```
>>> pp(sorted([str(x) for x in Path(git_root).glob('charts/**/*serviceaccount*.yaml')]))
[   'charts/astronomer/templates/commander/commander-serviceaccount.yaml',
    'charts/astronomer/templates/config-syncer/config-syncer-serviceaccount.yaml',
    'charts/astronomer/templates/houston/api/houston-bootstrap-serviceaccount.yaml',
    'charts/astronomer/templates/registry/registry-serviceaccount.yaml',
    'charts/elasticsearch/templates/es-serviceaccount.yaml',
    'charts/external-es-proxy/templates/external-es-proxy-serviceaccount.yaml',
    'charts/fluentd/templates/fluentd-serviceaccount.yaml',
    'charts/grafana/templates/grafana-bootstrap-serviceaccount.yaml',
    'charts/kube-state/templates/kube-state-serviceaccount.yaml',
    'charts/nginx/templates/nginx-default-backend-serviceaccount.yaml',
    'charts/nginx/templates/nginx-serviceaccount.yaml',
    'charts/pgbouncer/templates/pgbouncer-serviceaccount.yaml',
    'charts/postgresql/templates/serviceaccount.yaml',
    'charts/prometheus-node-exporter/templates/serviceaccount.yaml',
    'charts/prometheus-postgres-exporter/templates/serviceaccount.yaml',
    'charts/prometheus/templates/prometheus-serviceaccount.yaml']
```

Reply:
okay let me include this change similar to what we are doing for other tests